### DISCUSSION TITLE
flux suddenly stopped working

### DISCUSSION THREAD

**Discussion by RLutsch** [fluxcd/flux2#3134] [General] [answered]
so my last rollout didn't work and realised that the source controller lost conact with github 

{"level":"error","ts":"2022-09-23T09:28:03.998Z","msg":"failed to checkout and determine revision: unable to clone 'ssh://git@github.com/mygithub: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none publickey], no supported methods remain","controller":"gitrepository","controllerGroup":"source.toolkit.fluxcd.io","controllerKind":"GitRepository","gitRepository":{"name":"flux-system","namespace":"flux-system"}

I updated the token and updated flux to latest again but still nothing

flux bootstrap github --owner=<myorg> --repository=infrastructure --branch=main --path=./clusters/production

**Comment by stefanprodan** [comment, marked as answer]
`flux bootstrap` does not override the deploy key if it exists, so delete the secret with `kubectl -n flux-system delete secret flux-system`, then rerun the bootstrap which will create a new deploy key in GitHub and in the cluster.

**Reply by RLutsch** [reply]
thanks for the quick response and saving my rear end!

**Reply by stefanprodan** [reply]
You're welcome 😄
